### PR TITLE
Fix/content readthedocs4

### PR DESCRIPTION
**Pull request by santiagoinfantinom**
4th and last part of the correction of the readthedocs

### CODE REVIEW

File: docs/Constrains.rst [hunk 1]
```diff
@@ -5,7 +5,7 @@ General: Adding constraints
 
 Sometimes, a scenario analyzed by oemof needs additional constraints for a proper optimization result. As presented in oemof's `readthedocs <https://oemof.readthedocs.io/en/stable/_modules/oemof/solph/constraints.html>`_ this could be as easy as limiting the total investment allowed. In other cases, more elaborate constraints have to be formulated.
 
-In the case of this tool, a constraint defining minimal grid-stabilizing capacities (fossil-fulled generator, storage) was needed. The constraint has to be full-filled in every time step, but also accesses constants and optimized investment capacities. The procedure to add this customized constraint shall be described below.
+In the case of this tool, a constraint defining minimal grid-stabilizing capacities (fossil-fulled generator, storage) was needed. The constraint has to be fullfilled in every time step, but also accesses (?) constants and optimized investment capacities. The procedure to add this customized constraint shall be described below.
```
Comment (santiagoinfantinom) [inline docs/Constrains.rst:8]: Needs to access constant values and optimized investment capacities?

File: docs/Constrains.rst [hunk 2]
```diff
@@ -100,7 +100,7 @@ To make sure this rule is applied to every timestep, you can either explicitly l
 Sidenotes: Indirectly accessing attributes of Flows of an oemof-model (1)
 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-Calling constant attributes of Flows or InvestmentFlows indirectly makes most sense, if a certain class of components/busses is subjected to the constraint and if you can not or do not want to group it's element. Possible classes can be:::
+Calling constant attributes of Flows or InvestmentFlows indirectly makes most sense, if a certain class of components/busses is subjected to the constraint and if you can not or do not want to group it's element (?). Possible classes can be:::
```
Comment (santiagoinfantinom) [inline docs/Constrains.rst:103]: This is a bit unclear

File: docs/Constrains.rst [hunk 3]
```diff
@@ -87,7 +87,7 @@ Make sure that this value is defined for your component.
 Equation for pyomo (3)
 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-The constraint itself is defined by an equation (<=, ==, >=). Make sure, that your equation does not result in a value (True/False) but in the generation of an commonly applicable rule. This can happen if you print out results during coding the constraint and use calls like _model.flow[el_bus,sink_demand,t].value_.::
+The constraint itself is defined by an equation (<=, ==, >=). Make sure, that your equation does not result in a value (True/False) but in the generation of an commonly applicable rule (?). This can happen if you print out results during coding the constraint and use calls like _model.flow[el_bus,sink_demand,t].value_.::
```
Comment (santiagoinfantinom) [inline docs/Constrains.rst:90]: This was not so clear. You mean the code should yield another function?

File: docs/Constrains.rst (hunk 2 above)
Comment (santiagoinfantinom) [reply docs/Constrains.rst:103]: @smartie2076

File: docs/Constrains.rst (hunk 1 above)
Comment (santiagoinfantinom) [reply docs/Constrains.rst:8]: @smartie2076

Comment (smartie2076) [reply docs/Constrains.rst:8]: And also influences the optimization result of the capacities to be installed.

You can delete the part about the constants.

File: docs/Constrains.rst (hunk 3 above)
Comment (smartie2076) [reply docs/Constrains.rst:90]: Weird, I think I answered this with my phone and the answer was not submitted o.O

Yes, basically a rule would be a function, which is an unequality.

File: docs/Constrains.rst (hunk 2 above)
Comment (smartie2076) [inline docs/Constrains.rst:103]: Hmm... change to this:

```suggestion
Calling constant attributes of Flows or InvestmentFlows indirectly by accessing the oemof-solph assets can be helpful, if only a certain class of components/busses is subjected to the constraint. Possible classes can be:::
```
( I am actually not quite clear on this)

**Review by santiagoinfantinom — COMMENTED** [review]
Last changes. Ready for review and merge

**Review by smartie2076 — APPROVED** [review]

**Pull request merged**